loader/plugins/prune.js

```diff
@@ -0,0 +1,42 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+module.exports = class HelpPlugin extends CordlrPlugin {
+  constructor (bot, config) {
+    super(bot, config)
+
+    this.name = 'Cordlr Core - Prune'
+    this.description = 'Cordlr Core plugin for pruning an specific amount of messages'
+
+    this.commands = {
+      'prune': {
+        'usage': '<x>',
+        'function': 'pruneMessages',
+        'description': 'Prunes the last x messages',
+        'permissions': [
+          'MANAGE_MESSAGES'

```

Comment:
I tried it, but it worked for me. Also tried to console log the promise rejection but didn't received anything.

I created a group for the bot with the "manage messages" permission and an admin group which was higher in hierarchy and set it to administrator.